fastparquet/dataframe.py

```diff
@@ -5,6 +5,7 @@
 from pandas.core.frame import DataFrame
 from pandas.core.index import RangeIndex, Index
 from pandas.core.categorical import Categorical, CategoricalDtype
+from pandas.api.types import is_categorical_dtype
```

Comment:
Does this need pandas_compat?

Reply:
@martindurant whats' the minimum required pandas version? `pd.pai.types.is_categorical_dtype` is available in 0.19.2, but not 0.18.1.

Reply:
I am open to recommendations... there is no explicit testing against pandas versions. We do have the various Index types, that's the last time I recall explicitly considering the pandas version.